builtins/amp-ad.js

```diff
@@ -227,7 +232,7 @@ export function installAd(win) {
       // it is visible.
       if (inViewport) {
         this.unlistenViewportChanges_ =
-            this.getViewport().onChanged(this.sendAdIntersection_.bind(this));
+            this.getViewport().onScroll(this.sendAdIntersection_.bind(this));
```

Comment:
Please correct if I'm wrong, but this will send an intersection signal on every scroll event? Is this what you want?